app/src/features/apiClient/screens/apiClient/components/clientView/components/request/RequestBody.tsx

```diff
@@ -31,6 +29,12 @@ function parseSingleModeBody(params: {
 
 const RequestBody: React.FC<RequestBodyProps> = (props) => {
   const { contentType, variables, setRequestEntry, setContentType } = props;
+  const [isRequestBodyFullScreen, setIsRequestBodyFullScreen] = useState(false);
+
+  const handleFullscreenChange = () => {
+    setIsRequestBodyFullScreen((prev) => !prev);
+  };
```

Comment:
This state is not used in this component and just passed to `RawBody` instead keep this inside `RawBody` component